src/components/calcite-accordion/calcite-accordion.stories.ts

```diff
@@ -5,6 +5,7 @@ import { iconNames } from "../../../.storybook/helpers";
 import { select, text } from "@storybook/addon-knobs";
 import accordionReadme from "./readme.md";
 import accordionItemReadme from "../calcite-accordion-item/readme.md";
+import simpleSvgPlaceholder from "../../utils/simpleSvgPlaceholder";
```

Comment:
If this is mostly for stories, this could be moved into `.storybook/utils.tsx`.

Reply:
@jcfranco I think stories and potentially test apps so maybe its better here?